packages/web-app-files/src/components/SideBar/Shares/Collaborators/RoleDropdown.vue

```diff
@@ -157,11 +156,8 @@ export default {
       return this.allowSharePermission && this.resource.canShare()
     },
     share() {
-      const userShares = this.sharesTree[this.resource.path]?.filter((s) =>
-        ShareTypes.containsAnyValue(ShareTypes.individuals, [s.shareType])
-      )
-
-      return userShares?.length ? userShares[0] : undefined
+      // the root share has an empty key in the shares tree. That's the reason why we retrieve the share by an empty key here
+      return this.sharesTree['']?.find((s) => s.incoming)
```

Comment:
But that's only true if we are in the share jail, right? For oc10 the old behaviour should be correct...

Reply:
i just checked it again, it's the same behavior for classic too 

![Bildschirmfoto 2022-07-06 um 12 04 31](https://user-images.githubusercontent.com/49308105/177526041-e8f6e4f4-337b-47a9-99b7-d7ec85651fc5.png)